src/shipit_code_coverage/shipit_code_coverage/codecov.py

```diff
@@ -169,6 +169,15 @@ def generate_chunk_mapping(self):
             tar.add('chunk_mapping.sqlite')
 
     def go(self):
+        if self.from_pulse:
+            commit_sha = self.githubUtils.get_commit(self.revision)
```

Comment:
Nit: Shouldn't you move the definition of `commit_sha` above the `if`, in order for it to still be defined in `self.githubUtils.post_github_status(commit_sha)` below?